elasticsearch-model/spec/elasticsearch/model/naming_inheritance_spec.rb

```diff
@@ -19,105 +19,185 @@
 
 describe 'naming inheritance' do
 
-  before(:all) do
-    class ::TestBase
-      extend ActiveModel::Naming
+  context 'without using proxy' do
+    before(:all) do
+      TestBase = Class.new do
```

Comment:
Had to change the way the classes were defined so that they can be redefined with different base classes in the other context.